src/Categories/Monad/Morphism.agda

```diff
@@ -0,0 +1,74 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Level
+open import Categories.Category
+open import Categories.Monad
+
+module Categories.Monad.Morphism {o ℓ e} {C D : Category o ℓ e} where
+
+open import Categories.NaturalTransformation
+open import Categories.Functor
+
+open NaturalTransformation
+
+-- monad morphism in the sense of the nLab
+-- https://ncatlab.org/nlab/show/monad#the_bicategory_of_monads
+-- between generic monads t : a -> a & s : b -> b
+record Monad⇒ (M : Monad C) (N : Monad D) : Set (o ⊔ ℓ ⊔ e) where
+
+  module M = Monad M
```

Comment:
I would tend to make all of these `private`, otherwise they all appear on the interface for `Monad⇒`